### PR TITLE
Terraform

### PR DESCRIPTION
**Pull request by pritchyspritch**
Terraform for nessus, install scripts

### CODE REVIEW

File: generate_api_keys.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+base_url = f"https://{nessus_ip}:8834"
+
+session_url = "/session"
+params = {
+    "username": nessus_username,
+    "password": nessus_password
+}
+print(nessus_username)
+print(nessus_password)
```
Comment (alice-carr) [inline generate_api_keys.py:31]: we should get rid of these prints

File: nessus.py [hunk 2]
```diff
@@ -53,7 +81,8 @@ def prepare_export(id=5):
     return response_dict
 
 
-def token_download(export_token):
+def token_download(export_token, custom_headers):
+    print(export_token)
```
Comment (alice-carr) [inline nessus.py:85]: another print we can remove

File: generate_api_keys.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+response = requests.post(base_url + session_url, data=params, verify=False)
+response_token = json.loads(response.text)
+print(f"response_token: {response_token}")
```
Comment (denizgenc) [inline generate_api_keys.py:34]: Should we get rid of these print statements?

File: terraform/cloudinit/nessus_instance.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    expect "Is that ok"
+    send -- "y\n"
+
+    expect "User added"
+
+    EOF
+  - 'chmod 0755 /usr/local/bin/nessus-cloudinit.sh'
+  - '/usr/local/bin/nessus-cloudinit.sh >> /var/log/nessus-cloudinit.log 2>&1'
+  - 'chmod 0755 /usr/local/bin/nessus-user.sh'
```
Comment (alice-carr) [inline terraform/cloudinit/nessus_instance.yml:56]: we call nessus-user.sh nessus-user.expect earlier

File: terraform/lambda.tf [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Lambda
```
Comment (alice-carr) [inline terraform/lambda.tf:1]: probably a redundant comment considering the file is called lambda.tf

File: terraform/data.tf [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  name = "/nessus/licence"
+}
+#
+# data "aws_ssm_parameter" "nessus_access_key" {
+#   name  = "/nessus/access_key"
+# }
+#
+# data "aws_ssm_parameter" "nessus_secret_key" {
+#   name  = "/nessus/secret_key"
+# }
```
Comment (denizgenc) [inline terraform/data.tf:19]: Consider removing these lines

File: terraform/lambda.tf [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  timeout          = "900"
+  memory_size      = 2048
+
+  environment {
+    variables = {
+      nessus_username = data.aws_ssm_parameter.nessus_username.value
+      nessus_password = data.aws_ssm_parameter.nessus_password.value
+    }
+    # nessus_access_key  = "${data.aws_ssm_parameter.nessus_access_key}"
+    # nessus_secret_key  = "${data.aws_ssm_parameter.nessus_secret_key}"
```
Comment (denizgenc) [inline terraform/lambda.tf:17]: We should consider removing these comments too

**Pull request merged**